ambuda/templates/dictionaries/index.html

```diff
@@ -3,6 +3,16 @@
 {% import 'macros/dictionaries.html' as m %}
 
 
+{% macro _check(name, title) %}
```

Comment:
Maybe call this `_checkbox`? Just to avoid confusion that `_check(...)` checks (verifies) something.